### DISCUSSION TITLE
Does the `icon` of `features` frontmatter support image path?

### DISCUSSION THREAD

**Discussion by zhongds** [web-infra-dev/rspress#1393] [Q&A]
![image](https://github.com/user-attachments/assets/aed57c12-9f3e-4554-940f-80cbff3ce074)

icon只展示string，有没有办法支持图片路径或者是`<img src="xxx" />`之类的写法

**Comment by Timeless0911** [comment]
https://github.com/web-infra-dev/rspress/pull/1395 After this PR merged, you can use html string like `'<img src="xxx" />'` here. This feature will ship in tomorrow's `nightly` version and the official version will ship next week.
